docs/multicluster/user-guide.md

```diff
@@ -47,13 +47,14 @@ To use the latest version of Antrea Multi-cluster from the Antrea main branch,
 you can change the YAML manifest path to: `https://github.com/antrea-io/antrea/tree/main/multicluster/build/yamls/`
 when applying or downloading an Antrea YAML manifest.
 
-Since Antrea 1.7.0, multi-cluster Services require an Antrea Multi-cluster
-Gateway to be set up in each member cluster to route Service traffic across
-clusters, and Service CIDRs (ClusterIP ranges) must not overlap among clusters.
-To support Multi-cluster Gateways, `antrea-agent` must be deployed with the
-`Multicluster` feature enabled in a member cluster. You can set the following
-configuration parameters in `antrea-agent.conf` of the Antrea deployment
-manifest to enable the `Multicluster` feature:
+[Multi-cluster Services](#multi-cluster-service) and
+[multi-cluster Pod-to-Pod Connectivity](#antrea-multicluster-pod-to-pod-connectivity),
```

Comment:
I suggested to rename "antrea-multicluster-pod-to-pod-connectivity" to "multicluster-pod-to-pod-connectivity" in Jiajing's PR.

Reply:
Done